packages/backend/src/Metrics.ts

```diff
@@ -12,16 +12,52 @@ import {
 } from 'prom-client'
 
 export type RepositoryHistogram = Histogram<'repository' | 'method'>
+export type ProjectGauge = Gauge<'project'>
 
 export class Metrics {
   readonly repositoryHistogram: RepositoryHistogram
+  readonly activityLast: ProjectGauge
+  readonly activityLatest: ProjectGauge
+  readonly activityIncludedInApi: ProjectGauge
+  readonly activityConfig: Gauge<
+    'project' | 'scheduleIntervalMs' | 'batchSize' | 'uncertaintyBuffer'
+  >
 
   constructor() {
     this.repositoryHistogram = this.createHistogram({
       name: 'repository_method_duration_seconds',
       help: 'duration histogram of repository methods',
       labelNames: ['repository', 'method'],
     })
+
+    this.activityLast = this.createGauge({
+      name: 'activity_last_synced',
+      help: 'Last unit (block or day) synced by the sequence processor',
+      labelNames: ['project'],
+    })
+
+    this.activityLatest = this.createGauge({
+      name: 'activity_latest',
+      help: 'Latest existing unit to be synced to',
+      labelNames: ['project'],
+    })
+
+    this.activityIncludedInApi = this.createGauge({
```

Comment:
I don't know if there is some use case in Grafana for activityIncludedInApi One idea that comes to my mind is that there could be some graph to show when we excluded some activity from API but I do not know if it is useful information. Also, the only use case for activity config is to show the config in Grafana but I would assume that it is unnecessary. @krzkaczor @sz-piotr do you agree?

Reply:
@torztomasz We agreed to keep that. It's cheep to store it in Prometheus and will give us a retrospective view in the application config.